google-cloud-tools-plugin/ultimate/build.gradle

```diff
@@ -36,6 +36,7 @@ compileJava.dependsOn project(':google-cloud-tools-plugin').tasks.getByName("com
 dependencies {
     compile files(project(':google-cloud-tools-plugin').sourceSets.main.output)
     compile project(':google-cloud-tools-plugin:google-cloud-core')
+    compile project(':google-cloud-tools-plugin:stackdriver-debugger')
```

Comment:
is this dependency necessary (Ultimate -> debugger)?

Reply:
yep.  AppEngineServerInstance and others reference the debugger.

Reply:
JavaeeFrameworkSupportProviderTestCase

Reply:
let me see if I can move it to the test deps

Reply:
moved the dep to testCompile